<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassesForm</class>
 <widget class="QWidget" name="ClassesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(88, 86, 86);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnStuClose">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>6</y>
      <width>16</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_left">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_right">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_class">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>进入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_class1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_class2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_class3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
